.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
- [ ] I would recommend not to push `.env` or any `unnecessary files` into Github. As it makes the structure clumsy and also most of the time your secret would be there in the `.env` file which you don't want to share, instead you should put it into the `.gitignore` file.